yoti_python_sdk/attribute_issuance_details.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+from yoti_python_sdk import date_parser
+import base64
+
+
+class AttributeIssuanceDetails(object):
+    def __init__(self, data_entry):
+        self.__token = base64.b64encode(data_entry.issuance_token)
+        self.__expiry_date = date_parser.datetime_with_microsecond(
+            data_entry.issuing_attributes.expiry_date
+        )
+        self.__attributes = [a for a in data_entry.issuing_attributes.definitions]
```

Comment:
I think this needs to be list of dictionary (instead of `IssuingAttributes_pb2.Definition`)